contracts/rwa/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "rwa"
```

Comment:
what does `rwa` mean?

Reply:
Real World Asset .. anyway that contract should not be here ... i will delete it.